src/containers/Configure/Configure.css

```diff
@@ -1,6 +1,7 @@
-.configure-container {
+.background-container {
+  justify-content: center;
+  align-items: center;
   display: flex;
-  flex-direction: column;
   background-image: linear-gradient(
```

Comment:
I think we should actually move this background property to a div container on the App component.  That way we don't have to reproduce the rule for each individual page

Reply:
I'll do that now, thanks! Keeping our code DRY